modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -27,75 +28,191 @@ UnitTest.asynctest('browser.tinymce.core.content.EditorContentTest', (success, f
     return htmlSerializer.serialize(node);
   };
 
+  let events = [];
+  const logEvent = (event: EditorEvent<{}>) => {
+    events.push(event.type);
+  };
+
+  const sAssertEventsFiredInOrder = (expectedEvents?: string[]) => Step.sync(() => {
```

Comment:
```suggestion
  const defaultExpectedEvents = [ 'beforesetcontent', 'setcontent', ... ];
  const sAssertEventsFiredInOrder = (expectedEvents: string[] = defaultExpectedEvents) => Step.sync(() => {
```
Maybe something like this instead? That way the assertion should become easier to read